blocks/columns/columns.css

```diff
@@ -30,4 +30,21 @@
     flex: 1;
     order: unset;
   }
+
+  .blog-authors.columns {
```

Comment:
The convention that we follow in CSS when styling blocks is: block name followed by block variant (`.columns.blog-authors` - `.${block name}.${block variant}` , following the order that authors put in the word document `Columns (Blog Authors)`.

This helps when the CSS grows to follow the stylings for individual variants.